src/cc/frontends/clang/b_frontend_action.cc

```diff
@@ -790,8 +790,27 @@ bool BTypeVisitor::VisitVarDecl(VarDecl *Decl) {
         return false;
       }
 
+      int ret = -1;
+      if (bpf_create_map_cb) {
+        struct bpf_create_map_args args;
+
+        args.type = map_type;
+        args.name = (char *)table.name.c_str();
+        args.key_size = table.key_size;
+        args.value_size = table.leaf_size;
+        args.max_entries = table.max_entries;
+        args.map_flags = table.flags;
+
+        /* TODO: Get return value and store in table.fd */
```

Comment:
This is a bit broken and wont fly upstream - did you even try to fix it? 

Can `bpf_create_map_cb` return <= 0 ? If yes, then the below is completely broken because it'll result in a call to `bpf_create_map` for remotes which I don't think makes any sense